nav2_dwb_controller/dwb_controller/src/dwb_controller.cpp

```diff
@@ -70,6 +71,7 @@ DwbController::followPath(const nav2_tasks::FollowPathCommand::SharedPtr command
         RCLCPP_INFO(get_logger(), "No pose. Stopping robot");
         publishZeroVelocity();
       } else {
+        checkRegion(pose2d);
```

Comment:
The return code isnt used and DWB compute velocity commands will look at the correct time projection for collisions

Reply:
Agree, please review my comment above.